manifests/init.pp

```diff
@@ -34,13 +34,13 @@
 #                               type:Optional[Stdlib::Absolutepath]
 #
 # $forward::                    The forward option
-#                               type:Optional[String]
+#                               type:Optional[Enum['only', 'first']]
```

Comment:
since the introduction of types requires a major version bump we could just switch all to quoted?